Releases/Changelog.md

```diff
@@ -4,6 +4,12 @@ tags: [release]
 
 <!-- Use emojis from https://gitmoji.dev/ -->
 
+## next
+
+**to be released**
+
+- 💄 **Icons**: Replaced Feather and Tabler icons with Lucid icons. You can now select any Lucid icon for additional links in the side panel.
```

Comment:
Just noticed a little typo here, it should be "Lucide" with an e at the end, not just "Lucid" :)

Reply:
Ups, thanks it's fixed on `main`.